### PR TITLE
Small changes towards feed-forward implementation

### PR DESCRIPTION
**Pull request by Peque**

### CODE REVIEW

File: speed.c [hunk 1]
```diff
@@ -59,8 +58,7 @@ static float _calculate_search_linear_speed(float force)
  */
 void set_linear_speed_variables(float force, bool run)
 {
-	linear_acceleration = 2 * force / MOUSE_MASS;
-	linear_deceleration = 2 * force / MOUSE_MASS;
+	max_force = force;
```
Comment (cua-cua) [inline speed.c:65]: - I would take out the force set, and keep just max_linear_speed. The function name would be then 'set_max_linear_speed'. To update max_force I would always use 'set_max_force' as for calibration or move files. 'set_linear_speed_variables' is only used on main, so there I would add also set_max_force.

Comment (cua-cua) [reply speed.c:65]: - If you do not agree and want to keep the function as it is, change the docstring to add max_force.

**Review by cua-cua — CHANGES_REQUESTED** [review]

File: speed.c (hunk 1 above)
Comment (Peque) [reply speed.c:65]: I changed the name of the function and its description instead. How do you like that? :blush:

Comment (cua-cua) [reply speed.c:65]: perfect :+1:

**Review by cua-cua — APPROVED** [review]

**Pull request merged**